graph/path/bellman_ford_moore_test.go

```diff
@@ -170,6 +170,41 @@ func TestBellmanFordAllFrom(t *testing.T) {
 				}
 			}
 
+			paths = paths[:0]
+			pt.AllToFunc(test.Query.To().ID(), func(path []graph.Node) {
+				paths = append(paths, append([]graph.Node(nil), path...))
+			})
+			if weight != test.Weight {
+				t.Errorf("%q %s: unexpected weight from AllTo: got:%f want:%f",
+					test.Name, tg.typ, weight, test.Weight)
+			}
```

Comment:
why do we re-test this ?
didn't we test it at `L142` ?

(I am not advocating for removing it, mind you. I am just trying to see whether I am missing something)

Reply:
Yeah, this can go away.